### PR TITLE
Release defmt v0.3.6 (and others)

### PR DESCRIPTION
**Pull request by Urhengulas**
Release order:
- defmt-parser 0.3.4
- defmt-macros 0.3.7
- defmt 0.3.6
- defmt-decoder 0.3.10
- defmt-print 0.3.11

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.3.6] - 2024-02-05
+
```
Comment (BriocheBerlin) [inline CHANGELOG.md:11]: Doesn't the release PR itself (#808) need to go in here as well?

**Review by BriocheBerlin — COMMENTED** [review]
Thank you!

Not a problem of this PR, but maybe we can fix it here, it seem like https://github.com/knurling-rs/defmt/pull/787 is not showing up anywhere?

File: CHANGELOG.md (hunk 1 above)
Comment (Urhengulas) [reply CHANGELOG.md:11]: We sometimes did and sometimes don't. It does not add a lot of value to include it, but often I added it for completenes. Will add it here as well.

Comment (Urhengulas) [reply CHANGELOG.md:11]: Decided to not add it, to reduce noise.

**Pull request merged**